arm/Microsoft.Authorization/roleAssignments/parameters/parametersBicep.json

```diff
@@ -0,0 +1,15 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "roleDefinitionIdOrName": {
+      "value": "Owner"
+    },
+    "principalId":{
+      "value": "9fa1a3c1-d53b-40ea-8617-ec99e51285a3"
```

Comment:
obfuscate or remove

Reply:
Pending. Will setup a meeting to discuss.